moda-contracts/src/interfaces/ICatalogFactory.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+interface ICatalogFactory {
```

Comment:
I don't think we need this either. Orgs can deploy any contract they want, but we should review them before we register them

Reply:
Removed.